lib/hpc/utils.pm

```diff
@@ -21,11 +21,13 @@ our @EXPORT = qw(get_slurm_version);
 sub get_mpi() {
     my $mpi = get_required_var('MPI');
 
-    if ($mpi eq 'openmpi3') {
+    if ($mpi eq 'openmpi') {
```

Comment:
This wont work as it is now. 
All the tests use `openmpi3` and the above condition will never match. I assume that you propose to change the job groups accordingly. In this case, i would also recommend to rename the function(ex: get_openmpi_version)

But i wonder, as we  a job variable and now we tend to do the setup there, whether we need this function at all. it seems like its existence is a historical remnant and things have change in a degree which this approach is not necessary unless we really want to control the variable like this. But then we will have to update the function anyway with any new version. 

Did you think the case of completely removal of this function? wdyt?

Reply:
of course. I would change the test setting to openmpi